rspace/src/main/scala/coop/rchain/rspace/Hash.scala

```diff
@@ -0,0 +1,18 @@
+package coop.rchain.rspace
+
+import coop.rchain.crypto.hash.Blake2b256
+
+class Hash private (val bytes: Array[Byte])
+
+object Hash {
+
+  def create(bytes: Array[Byte]): Hash =
```

Comment:
Do you think it would be worth exposing an API that allows for choosing hashing function?

Reply:
This is really for internal use in `rspace` and we have no need to choose between hashing functions there.  Eventually we'll likely make this `private[rspace]` but I want to leave access unmodified for this development cycle.  I will add a TODO to that effect

I could call it `Blake2b256Hash`.